generatedTypes/components/badge/index.d.ts

```diff
@@ -344,7 +344,7 @@ declare const _default: React.ComponentClass<ViewProps & TouchableOpacityProps &
     /**
      * the badge size (default, small)
      */
-    size: number;
+    size?: number | undefined;
```

Comment:
Why do you need to specify undefined type for an optional prop?

Reply:
I don't, this is the automatically generated `.d.ts` file (when running `npm run build:dev`).